husky_components/dropdown/main.js

```diff
@@ -291,7 +337,25 @@ define([], function() {
         },
 
         // hide dropDown
-        hideDropDown: function() {
+        hideDropDown: function(event) {
+            // ignore it if the target is children of trigger but not children of item
+            if (!!event
+                && ((
+                    $(event.target).hasClass('husky-dropdown-trigger')
+                    || $(event.target).parents().hasClass('husky-dropdown-trigger')
+                )
+                && !(
+                    $(event.target).hasClass('husky-dropdown-item')
+                    || $(event.target).parents().hasClass('husky-dropdown-item')
+                )
+                || $(event.target).hasClass('highlight-animation')
+                || $(event.target).parents().hasClass('highlight-animation'))
+            ) {
+                this.sandbox.dom.one(this.sandbox.dom.window, 'click', this.hideDropDown.bind(this));
```

Comment:
Looks like the comment above is wrong, it does not simply ignore this case. (Although I don't really understand what it does at all).